test/Microsoft.AspNetCore.SignalR.Tests.Utils/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.AspNetCore.SignalR.Client.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
```

Comment:
Hmmm? Since it is just test related code, why not make it public?

Reply:
It was late ;P

Reply:
Oh, I remember why. It's because the `internal` type comes from a linked file that is in our product code. It has to be internal there. We could `#ifdef` the accessibility modifier I suppose but that gets gross.